internal/services/kusto/kusto_eventgrid_data_connection_resource_test.go

```diff
@@ -186,8 +188,8 @@ resource "azurerm_kusto_eventgrid_data_connection" "test" {
   blob_storage_event_type = "Microsoft.Storage.BlobRenamed"
   skip_first_record       = true
 
-  database_routing_type = "Multi"
-  eventgrid_resource_id = azurerm_eventgrid_event_subscription.test.id
+  database_routing_type           = "Multi"
+  eventgrid_event_subscription_id = azurerm_eventgrid_event_subscription.test.id
```

Comment:
We need to still include a test for the deprecate property to ensure we haven't broken anything with the changes and to ensure it still works until removal.

Reply:
Added tests:
- `TestAccKustoEventGridDataConnection_systemAssignedIdentityResourceId()`
- `TestAccKustoEventGridDataConnection_userAssignedIdentityResourceId()`